### DISCUSSION TITLE
Log into Postal mail account with IMAP?

### DISCUSSION THREAD

**Discussion by jonathonbyrdziak** [postalserver/postal#1765] [Help with using Postal]
Hey Guys,

Just a question here. I can't seem to find any documentation that talks about Postal and imap. 

I'd like to know if it's possible to have Postal hold the mail on the server until I log into it with my desktop application to download using imap.

If so, how would I go about that configuration and where do I configure the imap settings for Postal?

**Comment by willpower232** [comment]
Postal is not a traditional email server, there are no inboxes so no need for IMAP.

If you want to receive email from Postal then you'll need to set up something separately to process messages coming from Postal but it sounds like you want something else.
